builtin/logical/pki/config_util.go

```diff
@@ -44,9 +45,17 @@ func (sc *storageContext) updateDefaultIssuerId(id issuerID) error {
 	}
 
 	if config.DefaultIssuerId != id {
-		return sc.setIssuersConfig(&issuerConfigEntry{
+		err := sc.setIssuersConfig(&issuerConfigEntry{
 			DefaultIssuerId: id,
 		})
+
+		if err != nil {
+			return err
+		}
+
+		// Can we assume that this fetch won't raise an error?
```

Comment:
Hmmm... I'd prefer not to. Easy enough to return the err if it does occur. Otherwise, we'd get a nil issuer pointer to try to set the last modified timestamp at... The entry was already written so the main change will persist either way.